### PR TITLE
[FLINK-9508][Docs]General Spell Check on Flink Docs

### PR DESCRIPTION
**Pull request by medcv**
## What is the purpose of the change

- Spell check for Flink docs

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/dev/execution_configuration.md [hunk 1]
```diff
@@ -45,41 +45,41 @@ The following configuration options are available: (the default is bold)
 - **`enableClosureCleaner()`** / `disableClosureCleaner()`. The closure cleaner is enabled by default. The closure cleaner removes unneeded references to the surrounding class of anonymous functions inside Flink programs.
 With the closure cleaner disabled, it might happen that an anonymous user function is referencing the surrounding class, which is usually not Serializable. This will lead to exceptions by the serializer.
 
-- `getParallelism()` / `setParallelism(int parallelism)` Set the default parallelism for the job.
+- `getParallelism()` / `setParallelism(int parallelism)`. Set the default parallelism for the job.
```
Comment (zentol) [inline docs/dev/execution_configuration.md]: The period doesn't make sense as it isn't a sentence.

File: docs/internals/ide_setup.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ IntelliJ supports checkstyle within the IDE using the Checkstyle-IDEA plugin.
 3. Set the "Scan Scope" to "Only Java sources (including tests)".
 4. Select _8.4_ in the "Checkstyle Version" dropdown and click apply. **This step is important,
    don't skip it!**
-5. In the "Configuration File" pane, add a new configuration using the plus icon:
+5. In the "Configuration File" page, add a new configuration using the plus icon:
```
Comment (zentol) [inline docs/internals/ide_setup.md]: this is not a typo

File: docs/ops/filesystems.md [hunk 3]
```diff
@@ -70,21 +70,20 @@ That way, Flink seamlessly supports all of Hadoop file systems, and all Hadoop-c
...
 
-
 ## Common File System configurations
 
 The following configuration settings exist across different file systems
 
 #### Default File System
 
-If paths to files do not explicitly specify a file system scheme (and authority), a default scheme (and authority) will be used.
+If path to files do not explicitly specify a file system scheme (and authority), a default scheme (and authority) will be used.
```
Comment (zentol) [inline docs/ops/filesystems.md]: not a typo

File: docs/ops/filesystems.md [hunk 4]
```diff
@@ -70,21 +70,20 @@ That way, Flink seamlessly supports all of Hadoop file systems, and all Hadoop-c
...
 
 {% highlight yaml %}
 fs.default-scheme: <default-fs>
 {% endhighlight %}
 
-For example, if the default file system configured as `fs.default-scheme: hdfs://localhost:9000/`, then a a file path of
-`/user/hugo/in.txt'` is interpreted as `hdfs://localhost:9000/user/hugo/in.txt'`
+For example, if the default file system configured as `fs.default-scheme: hdfs://localhost:9000/`, then a file path of
+`'/user/hugo/in.txt'` is interpreted as `'hdfs://localhost:9000/user/hugo/in.txt'`
```
Comment (zentol) [inline docs/ops/filesystems.md]: I suggest removing the `'`  instead,

File: docs/ops/security-ssl.md [hunk 5]
```diff
@@ -22,22 +22,22 @@ specific language governing permissions and limitations
...
+SSL can be enabled for all network communication between Flink components. SSL keystores and truststore has to be deployed on each Flink node and configured (conf/flink-conf.yaml) using keys in the `security.ssl.*` namespace (Please see the [configuration page](config.html) for details). SSL can be selectively enabled/disabled for different transports using the following flags. These flags are only applicable when `security.ssl.enabled` is set to true.
 
 * **taskmanager.data.ssl.enabled**: SSL flag for data communication between task managers
 * **blob.service.ssl.enabled**: SSL flag for blob service client/server communication
-* **akka.ssl.enabled**: SSL flag for the akka based control connection between the flink client, jobmanager and taskmanager 
-* **jobmanager.web.ssl.enabled**: Flag to enable https access to the jobmanager's web frontend
+* **akka.ssl.enabled**: SSL flag for akka based control connection between the Flink client, JobManager and TaskManager
```
Comment (zentol) [inline docs/ops/security-ssl.md]: we typically write `jobmanager` and `taskmanager`, please revert

File: docs/ops/upgrading.md [hunk 6]
```diff
@@ -183,15 +183,15 @@ In this step, we update the framework version of the cluster. What this basicall
 the Flink installation with the new version. This step can depend on how you are running Flink in your cluster (e.g. 
 standalone, on Mesos, ...).
 
-If you are unfamiliar with installing Flink in your cluster, please read the [deployment and cluster setup documentation]({{ site.baseurl }}/ops/deployment/cluster_setup.html).
+If you are unfamiliar with installing Flink in your cluster, please read the [clusters and deployment setup documentation]({{ site.baseurl }}/ops/deployment/cluster_setup.html).
```
Comment (zentol) [inline docs/ops/upgrading.md]: this is an unnecessary change.

File: docs/ops/upgrading.md [hunk 7]
```diff
@@ -172,7 +172,7 @@ First major step in job migration is taking a savepoint of your job running in t
 You can do this with the command:
 
 {% highlight shell %}
-$ bin/flink savepoint :jobId [:targetDirectory]
+$ ./bin/flink savepoint <jobId> [savepointDirectory]
```
Comment (zentol) [inline docs/ops/upgrading.md]: `savepointDirectory` has specific semantics in the documentation and is assumed to be the directory with which one can start a job from again.
The directory passed here _does not fit these semantics_ as a `savepointDirectory` is created *within* the `targetDirectory`.

Please revert.

File: docs/ops/filesystems.md [hunk 8]
```diff
@@ -112,10 +111,9 @@ To prevent inactive streams from taking up the complete pool (preventing new con
 `fs.<scheme>.limit.stream-timeout`. If a stream does not read/write any bytes for at least that amount of time, it is forcibly closed.
 
 These limits are enforced per TaskManager, so each TaskManager in a Flink application or cluster will open up to that number of connections.
-In addition, the The limit are also enforced only per FileSystem instance. Because File Systems are created per scheme and authority, different
+In addition, the limit are also enforced only per FileSystem instance. Because File Systems are created per scheme and authority, different
```
Comment (zentol) [inline docs/ops/filesystems.md]: either `the limit is` or the `the limits are`

also, `only` should be placed before `enforced`.

**Comment by medcv** [comment]
@zentol Thanks a lot for review and I am so sorry for causing pain. I will reduce the scope of this PR and only keep spelling issues for now.

File: docs/internals/ide_setup.md (hunk 2 above)
Comment (medcv) [reply docs/internals/ide_setup.md]: will revert back!

File: docs/ops/filesystems.md (hunk 3 above)
Comment (medcv) [reply docs/ops/filesystems.md]: will revert back

File: docs/ops/filesystems.md (hunk 4 above)
Comment (medcv) [reply docs/ops/filesystems.md]: will do

File: docs/ops/security-ssl.md (hunk 5 above)
Comment (medcv) [reply docs/ops/security-ssl.md]: will revert back! I was reading other pages and seems there is some inconsistency on other pages as they used `JobManager` and `TaskManager` fromt

File: docs/ops/upgrading.md (hunk 6 above)
Comment (medcv) [reply docs/ops/upgrading.md]: will revert back! I waned to make it similar to the `Clusters and Deployment` page title

File: docs/dev/execution_configuration.md (hunk 1 above)
Comment (medcv) [reply docs/dev/execution_configuration.md]: Some of the items has period some doesn't. should we remove period for the ones with?

Comment (zentol) [reply docs/dev/execution_configuration.md]: I would remove them.

**Comment by medcv** [comment]
@zentol PR is updated! I would appreciate if you review and again so sorry for mixing concerns in a PR.
